pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.ruff.format]
+# https://docs.astral.sh/ruff/settings/#format
+line-ending = "lf"  # Use UNIX `\n` line endings for all files
+
+[tool.ruff.lint]
+# https://docs.astral.sh/ruff/rules/
+select = [
+    "E",    # pycodestyle errors
+    "F",    # pyflakes
+    "I",    # isort
+    "NPY",  # numpy
+    "PL",   # pylint
+    "UP",   # pyupgrade
+    "W",    # pycodestyle warnings
+]
```

Comment:
Let me know if these rules from https://docs.astral.sh/ruff/rules are too much or too little!

Reply:
Looks good to me for now. We can always adapt if we want more or someone feels slowed down by any of the linters.